src/app/components/filebrowseruss/filebrowseruss.component.ts

```diff
@@ -129,10 +146,23 @@ export class FileBrowserUSSComponent implements OnInit, OnDestroy {//IFileBrowse
     }
   }
 
+  browsePath(path: string): void {
+    this.path = path;
+  }
+
+  getDOMElement(): HTMLElement {
+    return this.elementRef.nativeElement;
+  }
+
+  getSelectedPath(): string {
+    //TODO:how do we want to want to handle caching vs message to app to open said path
+    return this.path;
+  }
+
   loadUserHomeDirectory(): void {
     this.isLoading = true;
-    this.ussSrv.getUserHomeFolder()
-      .subscribe(
+    const observable: Observable<any> = this.ussSrv.getUserHomeFolder()
```

Comment:
Did you add those yourself or were these accidental? (Your FE might be slightly out of date)